src/openapi.json

```diff
@@ -0,0 +1,566 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "Philanthropy Data Commons API",
+    "description": "An API for a common data platform to make the process of submitting data requests to funders less burdensome for organizations seeking grants.",
+    "version": "0.0.3"
+  },
+  "components": {
+    "parameters": {
+      "canonical_field_label": {
+        "name": "canonical_field_label",
+        "description": "The unique name of a canonical PDC field.",
+        "in": "path",
+        "required": true,
+        "schema": {
+          "type": "string"
+        }
+      },
+      "opportunity_title": {
+        "name": "opportunity_title",
+        "description": "The unique name of an opportunity, RFC, potential award or grant, etc.",
+        "in": "path",
+        "required": true,
+        "schema": {
+          "type": "string"
+        }
+      },
+      "applicant_external_id": {
+        "name": "applicant_external_id",
+        "description": "The unique name of an applicant, e.g. email address.",
```

Comment:
External ID here would be more like Tax ID Number (we have to talk to @jim-mcgowan to figure out how we handle organizations that have multiple "branches" under the same TIN).